unity3d/Assets/Swrve/SwrveSDK/SwrveComponent.cs

```diff
@@ -62,14 +63,16 @@ public static SwrveComponent Instance
         }
     }
 
+    public static SwrveSDK SDK { get { return Instance._SDK; } }
```

Comment:
Why is this static now? We have a static instance of the component, which in turn has an instance of the SDK